docs/kaiju_plugin_guide.md

```diff
@@ -0,0 +1,245 @@
+# Kaiju Plugin Guide Beta
+
+
+Kaiju plugins provide server owners the ability to add additional project types to Kaiju. These plugins control the components available in the project, the code that is generated and the generated preview.
+
+## Usage
+
+To add a new plugin to kaiju, modify the kaiju-plugin.config.js in the node server.
+```js
+import TerraPlugin from 'terra-kaiju-plugin';
+import NewPlugin from 'new-plugin';
+
+const config = {
+  terra: {
+    name: 'Terra',
+    plugin: TerraPlugin,
+  },
+  newPlugin: {
+    name: 'New Plugin',
+    plugin: NewPlugin,
+  },
+};
+
+export default config;
+```
+
+The config object should be built in the following format:
+
+```js
+const config = {
+  // Required: The key is used to uniquely identify kaiju projects created using this plugin.
+  key: {
```

Comment:
I think this would make more sense to me to use 'awesome_plugin' or 'awesomePlugin' instead of 'key' here. If you do that and update the comment to read "This key ..." instead of "The key ..." I think it all works out.